com.unity.render-pipelines.high-definition/Runtime/Material/Decal/DecalUtilities.hlsl

```diff
@@ -261,3 +261,25 @@ DecalSurfaceData GetDecalSurfaceData(PositionInputs posInput, float3 vtxNormal,
 
     return decalSurfaceData;
 }
+
+DecalSurfaceData GetDecalSurfaceData(PositionInputs posInput, FragInputs input, inout float alpha)
+{
+    float3 vtxNormal = input.tangentToWorld[2];
+    DecalSurfaceData decalSurfaceData = GetDecalSurfaceData(posInput, vtxNormal, alpha);
```

Comment:
Does it work if you flip the tangentToWorld[2] instead? 

i.e

float3 vtxNormal = input.tangentToWorld[2];
#ifdef _DOUBLESIDED_ON
    float3 doubleSidedConstants = _DoubleSidedConstants.xyz;
#else
    float3 doubleSidedConstants = float3(1.0, 1.0, 1.0);
#endif

ApplyDoubleSidedFlipOrMirror(input, doubleSidedConstants);

DecalSurfaceData decalSurfaceData = GetDecalSurfaceData(posInput, vtxNormal, alpha);

?